pandas/core/shared_docs.py

```diff
@@ -324,4 +324,57 @@
 0  0.000000   1.000000
 1  1.000000   2.718282
 2  1.414214   7.389056
+
+It is possible to call transform on a GroupBy object:
```

Comment:
You can call transform on a GroupBy object.

Reply:
I have made the change.